### DISCUSSION TITLE
API call "GET" with Filter failing

### DISCUSSION THREAD

**Discussion by ariba932** [supabase/supabase#17404] [Questions] [answered]
Using the API call to the search querys like *... /portal_users/id.eq.2 "  Is returning all records on the table. Even is jagun is typed (".. /portal_users/id.iiss.2" - this also return status code 200 and all the table records. 

Using supabase on the cloud - free version. 

Building my app with it.

**Comment by GaryAustin1** [comment, marked as answer]
If you go to the API tab in the dashboard and click BASH at top right it will show you the url format for selects and filters on a table.  Your format does not look correct.
`https://ugxxxxxxxxxxxxxxxxxq.supabase.co/rest/v1/cat?id=eq.1`  is the format generated.   Note the ?
Also https://postgrest.org/en/stable/references/api/tables_views.html  Shows the REST API url formats.

**Reply by ariba932** [reply]
Thanks. 

Resolved. 

Change  ? Id.eq.1 to  ?id=eq.1&select=•

Using the direct guide as you prescribe.
